MediaManager/Plugin.MediaManager.Abstractions/IVideoPlayer.cs

```diff
@@ -1,9 +1,13 @@
 ﻿using System;
+using Plugin.MediaManager.Abstractions.Implementations;
+
 namespace Plugin.MediaManager.Abstractions
 {
     public interface IVideoPlayer : IPlaybackManager
     {
         IVideoSurface RenderSurface { get; }
         void SetVideoSurface(IVideoSurface videoSurface);
+        VideoAspectMode AspectMode { get; }
+        void SetAspectMode(VideoAspectMode aspectMode);
```

Comment:
I was trying to follow the "guidelines" we have in the other interfaces (see IMediaQueue)